.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+<!--
+  For work in progress use the GitHub draft pull request feature.
+-->
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+- [ ] this PR contains breaking changes!
+
+# Resolves / is related to
+
+<!-- Relates to #IssueNumber1, #IssueNumber2 -->
+<!-- Closes #IssueNumber3 -->
+<!-- Closes #IssueNumber4 -->
+
+# What is affected by this PR
+
+- [ ] OpenFaaS Function Exchange Format
+- [ ] Message Format
+- [ ] Documentation
```

Comment:
Maybe clarify that the documentation should happen in the docs repo (for everything that is not javadoc).